definitions/json/Categories/Categories-Multiples.json

```diff
@@ -0,0 +1,548 @@
+[
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "CategoryID": "C1",
+    "CategoryLabel": "Starting a Claim",
+    "DisplayOrder": 1,
+    "ParentCategoryID": ""
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "CategoryID": "C2",
+    "CategoryLabel": "Response to a Claim",
+    "DisplayOrder": 2,
+    "ParentCategoryID": ""
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "CategoryID": "C3",
+    "CategoryLabel": "Initial Consideration",
+    "DisplayOrder": 3,
+    "ParentCategoryID": ""
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "CategoryID": "C4",
+    "CategoryLabel": "Case Management",
+    "DisplayOrder": 4,
+    "ParentCategoryID": ""
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "CategoryID": "C5",
+    "CategoryLabel": "Withdrawal /Settled",
```

Comment:
```suggestion
    "CategoryLabel": "Withdrawal/Settled",
```

Reply:
Harpreet this is important because of how the mapping is used in et-sya-api for instance.

Reply:
Hey Tensay, this was for case file view rather than the application mappings but have removed the space anyway